test/tests/interfacekernels/InterfaceSorptionSievert/tests

```diff
@@ -7,6 +7,7 @@
     input = 'interface_sorption_Sievert.i'
     exodiff = 'interface_sorption_Sievert_out.e'
     abs_zero = 1e-7
+    mesh_mode = REPLICATED
```

Comment:
Could you also have a comment in the input file to specify that this is limited to replicated mesh.

Could you also, as @cticenhour mentioned in private discussions: note in the test spec (as a comment) that BreakMeshByBlockGenerator currently doesn't work with DISTRIBUTED.

Reply:
Done